utils/securityJobSummary.go

```diff
@@ -178,6 +178,10 @@ func RecordSarifOutput(cmdResults *Results) (err error) {
 	if err != nil || manager == nil {
 		return
 	}
+	if cmdResults.ExtendedScanResults == nil || !cmdResults.ExtendedScanResults.EntitledForJas {
```

Comment:
do we have an integration test to cover it? 
maybe an entitlement mock that'll return false - and making sure that in this case - there's no sarif?

Reply:
added